File: BBIS.Application/DTOs/ApplicationSetting/InstitutionDto.cs
namespace BBIS.Application.DTOs.ApplicationSetting
{
    public class InstitutionDto
    {
        public Guid? InstitutionId { get; set; }
        public string Name { get; set; }
        public string Address { get; set; }
        public string ContactNo { get; set; }
        public string EmailAddress { get; set; }
        public bool IsActive { get; set; }
    }
}

File: BBIS.Api/ClientApp/src/models/ApplicationSetting/InstitutionDto.ts
export interface IInstitutionDto {
    InstitutionId: Guid;
    Name: string;
    Address: string;
    ContactNo: string;
    EmailAddress: string;
    IsActive: boolean;
}

export class InstitutionDto implements IInstitutionDto {
    InstitutionId: Guid = '';
    Name: string = '';
    Address: string = '';
    ContactNo: string = '';
    EmailAddress: string = '';
    IsActive: boolean = true;
}